src/components/application_manager/include/application_manager/message.h

```diff
@@ -94,7 +94,7 @@ class Message {
   void set_correlation_id(int32_t id);
   void set_connection_key(int32_t key);
   void set_message_type(MessageType type);
-  void set_binary_data(BinaryData* data);
```

Comment:
Not certain if this can be deprecated directly, since the only difference is a `const` parameter. If it can't, you need to name the new function slightly differently.

Reply:
@jacobkeeler Done 2b29c82396c6b24e15f8d54405821f39de379741